### ISSUE TITLE
MMS with large resolution pictures not visible for recipients

### ISSUE THREAD

**Issue by bdvore** [moezbhatti/qksms#1299]
### DESCRIPTION
Hello, my friends are not able to see the pictures attached in my MMS. They receive the message but they are just able to see text and emojis and the picture download doesn't produce any visible error. The problem occurs whatever the compression rate is configured.

If I use the phone default messaging app, everything is working fine. So I tried to find differences on the sent pictures. And I've noticed that even with strong compression configured in QKSMS the picture resolution is always larger than what the default app produces.

Here are a few figures:

- QKSMS: 1067x1898 (2.0 MP) for 185 kB, 718x958 (0.7 MP) for 92 kB...
- default messaging app: 360x568 (0.2 MP) for 130 kB, 720x518 (0.4 MP) for 230 kB...

On my phone, the camera app launched by QKSMS only allows me to choose between the following resolutions:

- 23 MP (4:3)
- 20 MP (16:9)
- 8 MP (4:3)
- 8 MP (16:9)

But I'm facing the issue even with the smaller ones.

Is it possible that some picture resolution limits exists on mobile operator side or within other messaging apps?
If yes, it would be very interesting to have QKSMS able to automatically reduce the picture resolution with some predefined values for some ratios (4:3, 16:9...) with a toggle to enable/disable this feature from the configuration. 

Thank you. I really appreciate the theme configuration feature.

**Comment by bdvore** [comment]
Yes, I've tried with my wife's phone which is on the same carrier. And I don't have any issue to receive pictures.

I've also tried to send a picture previously resized (568x360, 0.2 MP) and compressed (less than 200 kB) but it didn't work. It seems that picture resolution is not related with my issue.

Finally I've just tried with [silence](https://f-droid.org/fr/packages/org.smssecure.smssecure/) and MMS are working fine with this app (with not-encrypted message), but I don't like it.

It seems that silence and QKSMS manage MMS sending differently. But I'm not an android developer and I'm not able to analyze both codes.
